istio-control/istio-discovery/templates/telemetryv2.yaml

```diff
@@ -122,5 +122,89 @@ spec:
                 code:
                   local:
                     inline_string: envoy.wasm.stats
+
+---
+{{- if .Values.telemetry.v2.sdfilter }}
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: stackdriver-filter
+  {{- if .Values.global.configRootNamespace }}
+  namespace: {{ .Values.global.configRootNamespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+spec:
+  configPatches:
+    - applyTo: HTTP_FILTER
+      match:
+        context: SIDECAR_OUTBOUND
+        listener:
+          filterChain:
+            filter:
+              name: "envoy.http_connection_manager"
+              subFilter:
+                name: "envoy.router"
+      patch:
+        operation: INSERT_BEFORE
+        value:
+          name: envoy.filters.http.wasm
+          config:
+            config:
+              root_id: stackdriver_outbound
+              configuration: |
+                {}
+              vm_config:
+                vm_id: stackdriver_outbound
+                runtime: envoy.wasm.runtime.null
+                code:
+                  local: { inline_string: envoy.wasm.null.stackdriver }
```

Comment:
This is not right for 1.4

https://github.com/istio/proxy/blob/release-1.4/extensions/stackdriver/testdata/stackdriver_filter.yaml

Please look at this for reference. @bianpengyuan

Reply:
do we need the local part? the master and release-1.4 has discrepancy here

Reply:
I would create two PRs, one for master another one for 1.4 because of the filter differences.